catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/RemoveCommand.java

```diff
@@ -45,8 +53,10 @@
 
     private static final String IDS_LIST_ARGUMENT_NAME = "IDs";
 
+    int batchSize = 250;
+
     @Argument(name = IDS_LIST_ARGUMENT_NAME, description = "The id(s) of the document(s) by which to filter.", index = 0, multiValued = true, required = false)
-    List<String> ids = null;
+    Set<String> ids = new HashSet<>();
```

Comment:
Might want to double check removal by ID in case it's not okay with a set here instead of a list